command/job/hunt.go

```diff
@@ -0,0 +1,182 @@
+package job
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+)
+
+func Hunt() error {
+	// create Nomad API client
+	nomadClient, err := api.NewClient(api.DefaultConfig())
+	if err != nil {
+		return err
+	}
+
+	// Get the jobs
+	jobs, _, err := nomadClient.Jobs().List(nil)
+	if err != nil {
+		return err
+	}
+
+	for _, job := range jobs {
+		if job.Type != "service" {
+			continue
+		}
+
+		// Get job's running allocations
+		jobAllocations, _, err := nomadClient.Jobs().Allocations(job.ID, true, nil)
+		if err != nil {
+			return err
+		}
+
+		var runningAllocationJobVersion []uint64
+		for _, allocation := range jobAllocations {
+			if allocation.ClientStatus == "running" {
+				runningAllocationJobVersion = append(runningAllocationJobVersion, allocation.JobVersion)
+			}
+		}
+
+		// check for job version discrepancy
+		for _, jobVersion := range runningAllocationJobVersion {
+			if jobVersion != runningAllocationJobVersion[0] {
+				shame(job.ID, jobAllocations)
+				break
+			}
+		}
+	}
+
+	return nil
+}
+
+func shame(jobID string, jobAllocations []*api.AllocationListStub) {
```

Comment:
lol @ func name but id rename it to something more appropriate ;)

Reply:
no o_O